src/main/java/ch/njol/skript/classes/data/JavaClasses.java

```diff
@@ -325,6 +326,36 @@ public String toVariableNameString(Quaternionf quaternion) {
 						return null;
 					}
 				}));
+
+		Classes.registerClass(new ClassInfo<>(UUID.class, "uuid")
+			.user("(uuid|universal(ly)? unique identifier)s?")
+			.name("UUID")
+			.description(
+				"A UUID is a universally unique identifier, which is a 128-bit number that is unique to each entity. "
+					+ "They are usually encoded as 36-character strings, which include 32 hexadecimal digits and "
+					+ "four hyphens, typically in this format: \"123e4567-e89b-12d3-a456-426614174000\"",
+				"In minecraft, it is used to identify entities (like players) in a way "
+					+ "that is unique across all entities, regardless of the entity's state.")
```

Comment:
This may be just me, but I feel this description is a little bit convoluted for the average Skripter.
Maybe make this sound a little simpler, and then include a wiki link: https://minecraft.wiki/w/UUID